doc/tutorials/content/openni_grabber.rst

```diff
@@ -36,13 +36,20 @@ the OpenNI Grabber.
   
   <iframe title="PCL OpenNI Viewer example" width="480" height="390" src="https://www.youtube.com/embed/x3SaWQkPsPI?rel=0" frameborder="0" allowfullscreen></iframe>
 
-So let's look at the code. From *visualization/tools/openni_viewer_simple.cpp*
+So let's look at the code. From *tools/openni_viewer_simple.cpp*
```

Comment:
This seems to have been changed a lot compared to the openni_viewer_simple.cpp. but I just updated the sleep method and left the rest alone, since its grown quite a lot in LOCs.